lib/cretonne/meta/cdsl/typevar.py

```diff
@@ -152,7 +228,7 @@ def __eq__(self, other):
             return False
 
     def __repr__(self):
-        # type: () -> str
+        # type: (TypeSet) -> str
```

Comment:
Strangely mypy checks on my machine succeed both with and without the self argument in the signature. I removed the self argument from the signatures of all added/modified code.